buildSrc/subprojects/performance-testing/src/main/groovy/gradlebuild/performance/tasks/PerformanceTest.groovy

```diff
@@ -49,10 +55,7 @@ import java.nio.charset.Charset
 abstract class PerformanceTest extends DistributionTest {
     public static final String TC_URL = "https://builds.gradle.org/viewLog.html?buildId="
     public static final Set<String> NON_CACHEABLE_VERSIONS = Sets.newHashSet("last", "nightly", "flakiness-detection-commit")
-    // Baselines configured by command line `--baselines`
-    private Property<String> configuredBaselines = getProject().getObjects().property(String.class)
-    // Baselines determined by determineBaselines task
-    private Property<String> determinedBaselines = getProject().getObjects().property(String.class)
+    private final Property<String> baselines = getProject().getObjects().property(String.class)
```

Comment:
Is this a comma-separated list? We should say so in Javadoc.

Also, can we use an abstract class with getters for the properties?

Reply:
Also, would it make things worse if we used a `ListProperty` here?

I think it's fine to follow up on any of this in a separate PR.